.github/workflows/prepare-release.yml

```
@@ -56,6 +56,14 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
+
+      - name: Set environment
+        # Convert version into staging deployment name (e.g 3.3.0 -> saleor-staging-v33)
+        run: |
+          set -x
+          environment=$(echo $VERSION | sed -n 's#\([0-9]\+\).\([0-9]\+\).*#saleor-staging-v\1\2#p')
```

Seems like a debug leftover?

```suggestion
        run: |
          environment=$(echo $VERSION | sed -n 's#\([0-9]\+\).\([0-9]\+\).*#saleor-staging-v\1\2#p')
```